### ISSUE TITLE
High-contrast mode: icon buttons with no text appear squashed

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#7448]
This is a bug specific to [Windows high-contrast mode](https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/). cc @kbayliss who did the testing for this.

Our buttons with icon only don’t render correctly in high-contrast mode:

![7448-1](https://user-images.githubusercontent.com/877585/129969264-48acc84e-142c-4d46-85aa-8f3d06a31e99.png)
![7448-2](https://user-images.githubusercontent.com/877585/129969270-f72b9073-cd6a-4bc9-ad89-18b6e3bc97e1.png)

This first screenshot of InlinePanel should have "move down" and "delete" buttons in place of the horizontal lines. Normal mode for reference:

![7448-3](https://user-images.githubusercontent.com/877585/129969425-ab6b2227-3749-4838-83b1-e486d326fe6b.png)

Places to update:

- InlinePanel (reorder x2  + delete)
- Button groups

**Comment by vupham04** [comment]
I can confirm that the issue is out of date. The issue should be closed?

When I tested for the InlinePanel on high-contrast mode, it shows the appropriate buttons:

![image](https://user-images.githubusercontent.com/46913707/145524265-af59d52a-6609-46e7-b1a9-8c1b88af9bb4.png)

**Comment by zerolab** [comment]
Closing this based on the above.
